openmc/plots.py

```diff
@@ -430,6 +429,19 @@ def meshlines(self, meshlines):
 
         self._meshlines = meshlines
 
+    @staticmethod
+    def check_color(err_string, color):
```

Comment:
Make this `_check_color` since a user is not expected to be calling this directly